### PR TITLE
Removed sending the entire collection of methodsCoveredByTest

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: admin-part/src/main/kotlin/Plugin.kt [hunk 1]
```diff
@@ -416,8 +416,6 @@ class Plugin(
             send(buildVersion, Routes.Build.Summary.Tests.All(it), coverageByTests.all)
             send(buildVersion, Routes.Build.Summary.Tests.ByType(it), coverageByTests.byType)
         }
-        //TODO remove after changes on the frontend
-        send(buildVersion, Routes.Build.CoveredMethodsByTest(buildRoute), methodsCoveredByTest)
```
Comment (ViktoriiaLebedeva) [inline admin-part/src/main/kotlin/Plugin.kt:420]: Also delete CoveredMethodsByTest from the Routes

File: admin-part/src/main/kotlin/Config.kt [hunk 2]
```diff
@@ -18,7 +18,7 @@ package com.epam.drill.plugins.test2code
 internal class RuntimeConfig(private val pluginId: String) {
     val realtime: Boolean = sysProp("features.realtime")?.toBoolean() ?: true
 
-    val realtimeCache: Boolean = sysProp("features.realtime.cache")?.toBoolean() ?: realtime
+    val realtimeCalculationCache: Boolean = sysProp("features.realtime.cache")?.toBoolean() ?: realtime
```
Comment (ViktoriiaLebedeva) [inline admin-part/src/main/kotlin/Config.kt:21]: why did you rename this?

**Review by ViktoriiaLebedeva — CHANGES_REQUESTED** [review]

File: admin-part/src/main/kotlin/Config.kt (hunk 2 above)
Comment (Mazel-Tovr) [reply admin-part/src/main/kotlin/Config.kt:21]: we discussed naming with a @LikhanovMaxim  and came to the conclusion that realtimeCache sounds 
not clear enough, I didn't have time to change it in the last commit, so I'll change it in this

File: admin-part/src/main/kotlin/Plugin.kt (hunk 1 above)
Comment (ViktoriiaLebedeva) [reply admin-part/src/main/kotlin/Plugin.kt:420]: You forgot to remove it from the Routes.Build

**Review by ViktoriiaLebedeva — CHANGES_REQUESTED** [review]

**Review by LikhanovMaxim — APPROVED** [review]

**Review by ViktoriiaLebedeva — APPROVED** [review]
